mne/evoked.py

```diff
@@ -402,10 +402,11 @@ def plot(self, picks=None, exclude='bads', unit=True, show=True, ylim=None,
             channel traces will not be shown.
         window_title : str | None
             The title to put at the top of the figure window.
-        spatial_colors : bool
-            Color code lines by mapping physical sensor coordinates into color
-            values. Spatially similar channels will have similar colors.
-            Bad channels will be dotted.
+        spatial_colors : None | 'head' | 'skirt' | 'notopo'
```

Comment:
None | bool | str
[If None/False, no spatial colors.]
If True or 'skirt', plot spatial colors and legend topomap with 'skirt'.
If 'head', plot spatial colors and legend topomap with 'head'.
If 'notopo', plot spatial colors, but no legend.

Another thing: if we use the spatial_colors param like this, we can't for example use it to set a different color scheme ...